.cloud-build/execute_changed_notebooks_cli.py

```diff
@@ -129,26 +138,34 @@ def str2bool(v):
     base_branch=args.base_branch,
 )
 
+
+results_file = f"gs://{args.artifacts_bucket}/{args.build_id}"
```

Comment:
Simplify to `results_file = f"gs://{args.artifacts_bucket}/{args.build_id}.csv"`

Reply:
done